src/js/components/sharedComponents/TopFive.jsx

```diff
@@ -69,34 +77,49 @@ const TopFive = (props) => {
                 }}
                 onClick={(e) => getSelectedLink(e, result)}>
                 {linkText}
-            </a>];
+            </a>
+        ];
+
+        // TODO: remove once award links get added to Recipient Page
```

Comment:
this code block is in this file twice, are both needed?

Reply:
Both corresponding if statements will need to be removed in order to get the columns and rows back on the table